internal/config/cataloger_options.go

```diff
@@ -11,11 +11,13 @@ import (
 type catalogerOptions struct {
 	Enabled  bool         `yaml:"enabled" json:"enabled" mapstructure:"enabled"`
 	Scope    string       `yaml:"scope" json:"scope" mapstructure:"scope"`
+	Golang   golang       `yaml:"golang" json:"golang" mapstructure:"golang"`
```

Comment:
these are meant for options that apply to all catalogers, and also gives a pretty odd env var path to give: ``SYFT_PACKAGE_CATALOGER_GOLANG_...` .

I think there is another compromise here that elevates the importance of simplifying the user-facing values, probably trading off for short-term complexity in the config processing path.